docs/docs/dsl.md

```diff
@@ -36,6 +37,8 @@ We'll need the following imports to get started:
 import cats.effect._
 import cats.syntax.all._
 import org.http4s._, org.http4s.dsl.io._, org.http4s.implicits._
+import io.circe.generic.auto._
+import org.http4s.circe.CirceEntityCodec.* 
```

Comment:
Actually, it's bikeshedding (It won't fix the CI utterly), but we build docs with Scala 2, so you need the old-style import here